techniques/general/control-orientation.html

```diff
@@ -1,30 +1,31 @@
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to allow users to access content in the way the user prefers. A content provider may expect that most users will view content using a specific device orientation or may expect that a user will want to maintain the original view within the device. As a result the provider then prevents the content from rotating. By providing a control to allow the user to rotate the content, someone who needs to use a particular orientation will be able to view the content in a comfortable manner. For example, a person who cannot hold the device and has it mounted attached to a wheelchair or bed.</p>
-      <p class="note">Many users will expect to use OS-level settings to prevent the orientation from changing. As a result, it is usually preferable to satisfy this Success Criterion by not restricting the content's ability to adapt to changes in orientation and allowing users to use the OS orientation lock when desired.</p>
+			<p>The objective of this technique is to allow users to override author orientation decisions and view content as they prefer. Authors may design content for a preferred orientation and expect that people will use this orientation when viewing their content, or the author may decide that a user wants to maintain the orientation with which they initiated viewing. As a result the author may prevent the content from rotating. However, this fails the Orientation Success Criterion, since the author has restricted the view and operation to one orientation.</p>
+			<p>By providing a prevalent mechanism to allow the user to rotate restricted content, the author meets the Orientation requirement. For example, a person who cannot hold the device and has it mounted to a wheelchair or bed will be able to re-orient the content to match their display's physical orientation.</p>
+      <p class="note">Many users will expect to use OS-level settings to preserve a desired orientation. As a result, instead of restricting orientation at the content level, and then allowing a user to override that restriction, it is usually best to satisfy this Success Criterion by allowing users to enable	 the OS orientation lock when desired.</p>
     </section>
 		<section id="examples">
 			<h2>Examples</h2>
 			<section class="example">
-				<h3>An e-Reader application</h3>
-        <p>Users of an e-Reader web application sometimes read when laying down on their side. In this situation, the device is viewed by the user in portrait mode even though the device is in the landscape orientation with regard to gravity. The developer locks the content into the portrait orientation to allow for this reading behavior, but also provides a control to allow users who have devices mounted in a specific orientation to achieve the desired viewing orientation.</p>
+				<h3>A portrait-only e-Reader application with a rotate button</h3>
```

Comment:
Not going to fuss about this but prefer the simpler version from before

Reply:
meaning "Not going to fuss *any more beyond this*..." :)

Reply:
In regard to the "prevalent mechanism" I figured we'd end up with some discussion about this. Fine taking it out since it hasn't previously been discussed, but to me this is very similar to an existing requirement (can't think what one it is right now) where an alternative version of something is available and it must be prevalent. In the case of orientation, if someone had to dig through 3 layers of submenu while trying to read the screen sideways in order to rotate the page, I think most folks would say that is really not accomplishing the SC.

Reply:
BTW, I couldn't help thinking when I was reviewing this that it would be pretty odd to have an e-reader application that was web-based. I'm also not going to 'fuss anymore beyond this' either, but I did wish we could have come up with a more web-ish scenario.